en/mkdocs.yml

```diff
@@ -127,6 +127,7 @@ nav:
           - Use a Connection in Your Web Application: develop-components/sharing-and-reusing/use-a-connection-in-your-web-application.md
       - Manage Deployment Tracks for Choreo Components: develop-components/manage-deployment-tracks-for-choreo-components.md
       - Configure Endpoints: develop-components/configure-endpoints.md
+      - Source Configuration Descriptor Reference: develop-components/source-configuration-descriptor.md
```

Comment:
If we can go with "Manage Component Source Configurations" as the title, let's update the name here too and change the .md file name accordingly.

Reply:
Sure, Will update the title.